modules/nf-core/biscuit/bsconv/main.nf

```diff
@@ -22,19 +20,30 @@ process BISCUIT_BSCONV {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    if ("$bam" == "${prefix}.bam") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
     """
-    INDEX=`find -L ./ -name "*.bis.amb" | sed 's/\\.bis.amb\$//'`
-
-    biscuit bsconv \\
+    samtools \\
+        sort \\
```

Comment:
Not sure why this change was made. Samtools sort is not a replacement for biscuit bsconv